test/__fixtures__/activation/list-triggers-output.txt

```diff
@@ -0,0 +1,2 @@
+Datetime               Activation ID                    Kind    Start Duration Status  Entity                  
+11/27/2020, 9:35:19 AM a5e7fdaeaa2e4384a7fdaeaa2e438442 trigger --    --       success 8888_9999/trigger:0.0.1 
```

Comment:
this fails because of the actual time being different in different timezones

Reply:
Fair - do you have a suggestion to adapt the test?

Reply:
How about replacing the date after reading from the file and before comparing ?
```
const expDate = new Date(1606487719405)
let expOutput = fixtureFile('activation/list-triggers-output.txt')
expOutput = expOutput.replace('11/27/2020, 9:35:19 AM', expDate.toLocaleString())
return command.run()
        .then(() => {
          expect(stdout.output).toMatch(expOutput)
        })
```

Reply:
Thanks for the suggestions. Updated accordingly.